src/master/master_impl.cc

```diff
@@ -3801,6 +3805,11 @@ void MasterImpl::SplitTabletCallback(TabletPtr tablet,
             }
             node->FinishSplit(next_tablet);
         }
+    } else { // server down or restart
+        tablet->SetStatusIf(kTableOffLine, kTableOnSplit);
```

Comment:
i think it cannot return false，because other control flow ignore OnSplit tablet.